docs/customized-components/Logo.js

```diff
@@ -38,7 +38,7 @@ const styles = ({ color }) => ({
 
 export function LogoRenderer({ classes }) {
   return (
-    <a href="/design-system" className={classes.wrapper}>
```

Comment:
I think this should not get changed

Reply:
We have a bug related to this link so I wanted to fix it by the way :)